build.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# rmm build script
+
+# This script is used to build the component(s) in this repo from
+# source, and can be called with various options to customize the
+# build as needed (see the help output for details)
+
+# Abort script on first error
+set -e
+
+NUMARGS=$#
+ARGS=$*
+
+# Get root of git repository without assuming location of build.sh script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPT_DIR"
+REPODIR=$(git rev-parse --show-toplevel)
```

Comment:
I would remove the `cd` and the `git rev-parse` and just do
```
REPODIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
```
which is basically how all the other scripts do it, then get rid of SCRIPT_DIR altogether. Unless, is there a reason you had to do it that way?

Reply:
This is not very foolproof. See for example: https://stackoverflow.com/questions/59895/get-the-source-directory-of-a-bash-script-from-within-the-script-itself